ssh_ldap_pubkey/__init__.py

```diff
@@ -84,6 +84,9 @@ def connect(self):
             conn.network_timeout = conf.bind_timeout
             conn.timeout = conf.search_timeout
 
+            if conf.ssl == 'start_tls' and conf.uri.startswith('ldap://'):
```

Comment:
> Although I suppose if you've specified an ldaps:// uri AND specified ssl start_tls in the config file it could be argued that it's a misconfiguration and should fail?

Yes, it’d be better to fail than silently ignore the configured ssl type.